docs/source/cpp/streaming_execution.rst

```diff
@@ -113,12 +109,12 @@ Read Relations
 
  * The ``projection`` property is not supported and plans containing this
    property will be rejected.
- * The only supported read type is ``LocalFiles``.  Plans with any other
-   type will be rejected.
- * Only the parquet file format is currently supported.
+ * The only ``VirtualTable`` and ``ExtensionTable``read types are not supported.
```

Comment:
@westonpace should we add unsupported status as a warning for plans with Virtual Table?

Reply:
I think we catch this already.  This is currently checked here (in relation_internal.cc):

```
      if (!read.has_local_files()) {
        return Status::NotImplemented(
            "substrait::ReadRel with read_type other than LocalFiles");
      }
```

It might be nice to say "LocalFiles or NamedTable".

Reply:
Ah yes. That's better.